### ISSUE TITLE
Event Time shifts 6h earlier after the event is created.

### ISSUE THREAD

**Issue by eabedrapo** [GetTogetherComm/GetTogether#238]
**Actual results:** when an event is created its time is shown as scheduled 6h earlier than the intended time, so organizers are forced to edit the event settings to set the time again.

**Expected results:** the event should show the time that the organizer selected on creation.

**Steps to reproduce:**
1. Click on "Host a Get Together" button.
2. Fill in the event name and date.
3. Select the time in which the event is going to occur (e.g. 6PM).
4. Select the time in which the event is planned to end (e.g. 8PM).
<img width="501" alt="Set time on creation" src="https://user-images.githubusercontent.com/1670763/67163040-13579f00-f330-11e9-9f4d-9ac4ac9656fc.png">

5. Save the event.
6. Check if the time for the event is the chosen one.
<img width="731" alt="Time right after creation" src="https://user-images.githubusercontent.com/1670763/67163041-13579f00-f330-11e9-92ad-59839d92a5c6.png">

**Comment by craigmaloney** [comment]
I think this has to do with time zones. I've noticed a lot of members in our groups are set as UTC, so if the group has set their time zone then things can get weird.

**Comment by eabedrapo** [comment]
Yep, I think so... I have set myself and my groups to be in Chicago time... and I'm noticing the problem every time I create an event.

Also, I reported that another event wasn't showing in the map and @mhall119 discovered that somehow my event was set to be "in the past" (see #232).

**Comment by eabedrapo** [comment]
This is the event I used as example in the report: https://gettogether.community/events/2753/workshop-working-with-page-objects/

But it occurred with all the events I have scheduled.

My profile has Chicago's location and timezone, as well as the team and the locations I'm using for the events. And I'm having the problem every time I create an event, so I have to manually edit them after creation to correct the scheduled time.

Thanks!

**Comment by bremensaki** [comment]
I've reproduced this with a user account with no TZ preference set (defaulted to UTC) which sort of explains it, but the timezone for the event was implied to be explicitly set during the workflow by the Timezone selector of the meeting place, which was definitely set to Australia/Melbourne. The event time was moved forward 11 hours when viewed after, matching the UTC+11 difference for Melbourne.

It looks like the location's TZ is overridden by the user account's when saving the new event (or maybe team, haven't tested) TZ - which shouldn't be the case. If you then go back to edit the event details, despite no setting being visible for timezone, the specified location's TZ is then applied, not the user's.

So the bug appears to be that during event _creation_ the user account TZ is always used, and the location's only takes precedence in an edit. I haven't looked at any code to verify any of this yet though.
